src/Microsoft.AspNetCore.Identity.UI/IdentityBuilderUIExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.AspNetCore.Identity.UI;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using Microsoft.AspNetCore.Identity.UI.Services;
+
+namespace Microsoft.AspNetCore.Identity
+{
+    public static class IdentityBuilderUIExtensions
+    {
+        public static IdentityBuilder AddDefaultUI(this IdentityBuilder builder)
+        {
+            builder.Services.AddMvc()
```

Comment:
Actually now that I think about it, this might be a good use case for using the new `ConfigureOptions<TSetup>` that we added for 2.1:

https://github.com/aspnet/Options/blob/dev/src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs#L139

Basically you can write a single IdentityUIConfigure setup class that implements multiple IConfigure[Named]Options, so this method could just call that with the uber setup, and setup the identity cookie + add the EmailSender service

Reply:
Done